dist/index.html

```diff
@@ -8,6 +8,13 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;1,300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;500;600;800;1000&family=Open+Sans:ital,wght@0,300;1,300&display=swap" rel="stylesheet">
 <script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
```

Comment:
- [ ] [OPTIONAL] Kindly make the design responsive, as responsitivity is crucial in today's web development world. 😀

![mobile](https://user-images.githubusercontent.com/41163571/231535316-0ce7e90e-ce91-4ee1-8991-05070c55ee1a.gif)